build/.gitignore

```diff
@@ -1,6 +1,12 @@
 VC2010
 VC2015
 VC2015_32
+DF
+DF_unmodified
 DF_PATH.txt
 _CPack_Packages
 *.tar.*
+*.sln
+*.vcxproj*
+dfhack_setarch.txt
+ImportExecutables.cmake
```

Comment:
In diff views, it will say "No newline at end of file" in red text, and GitHub will display this indicator:
![image](https://user-images.githubusercontent.com/3719547/44941122-74f02f00-ad65-11e8-8af4-ccb7067ad4bb.png)
You still have this in three other files - depends/CMakeLists.txt, library/LuaApi.cpp, andlibrary/modules/Curl.cpp